app/helpers/ical_helper.rb

```diff
@@ -7,8 +7,8 @@ def icalendar(*events)
       cal.event do |item|
         item.summary     = event.name
         item.description = "#{event.description} #{event_url(event)}"
-        item.dtstart     = event.date
-        item.dtend       = event.end_date
+        item.dtstart     = event.date.utc
```

Comment:
Some background on why this is needed:
https://stackoverflow.com/questions/47274517/ical-dtstart-and-dtend-format-confusion-should-be-easy-to-answer#answer-47275021